From: Alexander Gerasiov <gq@debian.org>
Subject: Fix in manpage to prevent lintian warning.
 Lintian dislike too long lines in manpage, so I've added non-significal space
 in filter example.
Forwarded: not-needed

Index: ldapscripts/man/man5/ldapscripts.5
===================================================================
--- ldapscripts.orig/man/man5/ldapscripts.5	2011-03-21 16:32:22.000000000 +0300
+++ ldapscripts/man/man5/ldapscripts.5	2011-03-21 16:40:46.000000000 +0300
@@ -73,4 +73,4 @@
 .B "Additional information: value does not conform to assertion syntax".
 Setting slapd's debug level to 32 shows additional details :
 .B "get_ava: illegal value for attributeType uidNumber".
-This is *not* a bug : the ldapscripts tend to use the power of LDAP filters to easily find users (or groups) using either a uidNumber (numerical value) or a uid (string value). The following filter (used by ldapfinger(1)) will generate the above message if $_USER is a login : "(|(&(objectClass=posixAccount)(|(uid=$_USER)(uidNumber=$_USER)))(&(objectClass=posixGroup)(|(cn=$_USER)(gidNumber=$_USER))))" because filter part "(uidNumber=$_USER)" requires an integer but gets a string. You can mostly ignore those warnings.
+This is *not* a bug : the ldapscripts tend to use the power of LDAP filters to easily find users (or groups) using either a uidNumber (numerical value) or a uid (string value). The following filter (used by ldapfinger(1)) will generate the above message if $_USER is a login : "(|(&(objectClass=posixAccount)(|(uid=$_USER)(uidNumber=$_USER))) (&(objectClass=posixGroup)(|(cn=$_USER)(gidNumber=$_USER))))" because filter part "(uidNumber=$_USER)" requires an integer but gets a string. You can mostly ignore those warnings.
